src/test/librbd/journal/test_Entries.cc

```diff
@@ -196,6 +196,69 @@ TEST_F(TestJournalEntries, AioDiscard) {
   ASSERT_EQ(234U, aio_discard_event.length);
 }
 
+TEST_F(TestJournalEntries, AioDiscardWithPrune) {
+  REQUIRE_FEATURE(RBD_FEATURE_JOURNALING);
+
+  // The discard path can create multiple image extents (ImageRequest.cc) in the
+  // case where the discard request needs to be pruned and multiple objects are
+  // involved in the request. This test ensures that journal event entries are
+  // queued up for each image extent.
+
+  // Create an image that is multiple objects so that we can force multiple
+  // image extents on the discard path.
+  CephContext* cct = reinterpret_cast<CephContext*>(_rados.cct());
+  const int64_t object_size = 1ull << cct->_conf.get_val<uint64_t>("rbd_default_order");
```

Comment:
Why change unsigned (`uint64_t`, `ull`) to signed here?

Reply:
Oh, sorry, that's just sloppiness - just made it `auto`.